src/pages/[...params].js

```diff
@@ -46,9 +46,7 @@ const Fallback = () => {
       '?',
     );
 
-    return `${
-      'http://localhost:9999' || publicRuntimeConfig.legacyAppUrl
-    }${path}${queryString}`;
+    return `${publicRuntimeConfig.legacyAppUrl}${path}${queryString}`;
```

Comment:
If you want to use `http://localhost:9999`, you can add `.env.local` to `source/udb3-frontend` with:
```
NEXT_PUBLIC_LEGACY_APP_URL=http://localhost:9999
```
That should override the `NEXT_PUBLIC_LEGACY_APP_URL` set in the `.env` that gets set by the initscripts.